Tests/ActionTests/ActionTests.swift

```diff
@@ -500,7 +500,11 @@ class ActionTests: QuickSpec {
                     }
 
                     scheduler.scheduleAt(30) {
-                        trigger.onNext()
+						#if swift(>=3.2)
```

Comment:
Do we need the separate handling? Wouldn't `trigger.onNext(())` work across both ? 

also, nit: Indentation looks off here